src/main/java/org/graylog/collector/inputs/InputConfiguration.java

```diff
@@ -30,6 +34,8 @@
 import java.util.Set;
 
 public abstract class InputConfiguration implements Configuration {
+    private static final Logger log = LoggerFactory.getLogger(InputConfiguration.class);
```

Comment:
The whole collector project is using `log` for the logger constant. I kept it that way for consistency.